cinn/hlir/op/op_nn_test.cc

```diff
@@ -22,18 +22,60 @@
 #include "cinn/cinn.h"
 #include "cinn/common/target.h"
 #include "cinn/common/test_helper.h"
+#include "cinn/hlir/framework/graph_compiler.h"
 #include "cinn/hlir/framework/node.h"
 #include "cinn/hlir/framework/op.h"
 #include "cinn/hlir/framework/op_strategy.h"
 #include "cinn/hlir/op/use_ops.h"
 #include "cinn/hlir/pe/nn.h"
+#include "cinn/runtime/flags.h"
+
+DECLARE_bool(cinn_ir_schedule);
 
 namespace cinn {
 namespace hlir {
 namespace framework {
 
 using CCompute = std::function<std::shared_ptr<ir::Tensor>(const std::vector<ir::Tensor>)>;
 
+void GetLoweredFunc(const std::string test_name,
+                    const std::string func_name,
+                    const std::shared_ptr<OpImpl> &impl,
+                    std::vector<std::string> &input_names,
+                    const std::string &output_name,
+                    std::vector<ir::Tensor> &inputs,
+                    std::vector<common::CINNValue> &cinn_inputs,
+                    const Target &target,
+                    Module::Builder &builder) {
```

Comment:
这个函数是生成ir::Module的过程，可以叫LowerToModule或BuildModule，直接返回生成的Module；另外input_names引用传递会作为输出函数，这与预期不符，这里外部不再使用可用值传递。
```suggestion
ModuleL LoweredToModule(const std::string test_name,
                    const std::string func_name,
                    const std::shared_ptr<OpImpl> &impl,
                    std::vector<std::string> input_names,
                    const std::string &output_name,
                    std::vector<ir::Tensor> &inputs,
                    std::vector<common::CINNValue> &cinn_inputs,
                    const Target &target) {
```